### PR TITLE
Add snowmass loi document

### PR DESCRIPTION
**Pull request by gordonwatts**

### CODE REVIEW

File: _data/publications/snowmass2021-analysis-ecosystem-loi.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: "Snowmass 2021 Letter of Interest: "
+link: https://www.snowmass21.org/docs/files/summaries/CompF/SNOWMASS21-CompF5_CompF7_Gordon_Watts-136.pdf
+date: 2020-09-10
+citation: "G. Watts"
```
Comment (davidlange6) [inline _data/publications/snowmass2021-analysis-ecosystem-loi.yml]: @gordonwatts - our citation entries usually have more than just the author. Seems for Snowmass people are adding "Snowmass 2021 Letters of Interest"

Comment (gordonwatts) [reply _data/publications/snowmass2021-analysis-ecosystem-loi.yml]: Thanks - fixed that!

Comment (davidlange6) [reply _data/publications/snowmass2021-analysis-ecosystem-loi.yml]: I really meant the citation.. Eg, "G. Watts Snowmass 2021 Letter of Interest" seems to be what others are doing.

**Comment by gordonwatts** [comment]
Thanks!!

**Pull request merged**